core/templates/dev/head/base.js

```diff
@@ -97,7 +97,10 @@ oppia.controller('Base', [
       if (!$scope.sidebarIsShown) {
         $scope.sidebarIsShown = true;
         $scope.pendingSidebarClick = true;
-      }
+      } else {
+        $scope.sidebarIsShown = false;
+        $scope.pendingSidebarClick = false;
+	  }
```

Comment:
@Yeeons -- Indentation of this last closing brace is still wrong, though; it was correct before. I think you still have tabs in here.